src/managers/GuildMemberManager.js

```diff
@@ -205,6 +205,24 @@ class GuildMemberManager extends CachedManager {
     return data.reduce((col, member) => col.set(member.user.id, this._add(member, cache)), new Collection());
   }
 
+  /**
+   * Options used for listing guild members.
+   * @typedef {Object} GuildListMembersOptions
+   * @property {Snowflake} after  Limit fetching members to those with an id greater than the supplied id
```

Comment:
All parameters are optional:
```suggestion
   * @property {Snowflake} [after] Limit fetching members to those with an id greater than the supplied id
```

Reply:
They are optional on the types, made a mistake with the jsdoc